annotation-server/src/medications/medications.service.ts

```diff
@@ -167,6 +163,14 @@ export class MedicationsService {
         return (await this.medicationRepository.count()) > 0;
     }
 
+    async patch(patch: PatchBodyDto<Medication>): Promise<void> {
+        await Promise.all(
+            patch.map(({ id, ...update }) =>
+                this.medicationRepository.update(id, update),
+            ),
+        );
+    }
```

Comment:
At the moment no DTOs are validated (#258). Is it fine if I just add a note there about making this specific DTO validatable as well?

Reply:
If I saw that correctly, adding the validation pipeline (https://docs.nestjs.com/techniques/validation) to the modules, all controller DTOs are automatically validated. As this DTO is directly on request level, it should be included there. So I guess, no comment would be needed.

Reply:
As far as I understand, this wouldn't apply in this case though since there actually isn't any class validation and `PatchBodyDto` is just a generic Typescript type. Or is there some magic happening that I'm unaware of?